src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -316,14 +350,22 @@ const SkillsQuizStepper = ({ isStyleAutoSuggest }) => {
         <ModalDialog.Footer>
           <Stepper.ActionRow eventKey="skills-search">
             <Button variant="outline-primary" onClick={() => closeSkillsQuiz()}>
-              Cancel
+              <FormattedMessage
+                id="enterprise.skills.quiz.v1.cancel.button.label"
+                defaultMessage="Cancel"
+                description="Cancel button text on the skills quiz v1 page."
+              />
             </Button>
             <Stepper.ActionRow.Spacer />
             <Button
               disabled={!canContinueToRecommendedCourses}
               onClick={() => flipToRecommendedCourses()}
             >
-              Continue
+              <FormattedMessage
+                id="enterprise.skills.quiz.v1.continue.button.label"
+                defaultMessage="Continue"
+                description="Continue button text on the skills quiz v1 page to continue towards recommended courses."
```

Comment:
What will happen on continue? Can we add little bit more context in the description?